README.md

```diff
@@ -64,6 +64,17 @@ file or an HTML file.
 The new file should contain what Jekyll calls [front matter](https://jekyllrb.com/docs/frontmatter/), which is essentially
 YAML markup that lets you set options like the template, the permalink, and the title of the page.
 
+### Add Envoy Adopter Logo
+
+Envoy is happy to recognize projects that use the software in production. These are not intended to be an advertisement, but a show of support for the project. All supporting organizations will be listed in alphabetical order. To be added to our adopter list, you must meet these criteria:
+
+*   Be a participant and active contributor in the community, this can be contributing code, issues or sponsoring EnvoyCon
+*   Publicly disclose your usage of Envoy via a [talk](https://www.youtube.com/watch?v=4x5WjxAMvKY), [blog](https://monzo.com/blog/2019/04/03/deploying-envoy-proxy) [CNCF case study](https://www.cncf.io/newsroom/case-studies/?_sft_cstudies_project=envoy) or [social media](https://twitter.com/suhailpatel/status/1113425967144476672)
+*   Submit a high quality vector (SVG) logo
```

Comment:
@richarddli do we want to request the SVG has a particular size/ratio and is also grayscale like the others?

Reply:
Yes, will check on this.

Reply:
Ok, so SVGs scale pretty well, so it's really about height vs width ratio. So not sure how to put into the guideline, other than "We scale the height to 52pix so make sure your logo isn't too tall :-)"?

Reply:
OK can we update this text slightly to just talk about both the height ratio we expect as well as the fact that we need the logo to be grayscale?

Reply:
done